packages/core/.gitignore

```diff
@@ -1 +1,3 @@
-src/interfaces/common
\ No newline at end of file
+# Custom
```

Comment:
was this meant to say Common?

Reply:
no, just added # Custom in here because here `packages/react-native-otlp/.gitignore` I saw a bunch of other titles as comment, and then just copied/pasted the same in here. I can remove/tweak if you prefer

Reply:
ya just didn't seem particularly useful in this case, I think it comes in handy when there's a larger file and we want to put a header on different sections

Reply:
done, removed